frontend/scripts/tests/explore/explore-selectors.test.js

```diff
@@ -0,0 +1,43 @@
+import { getCardsWithRecentCard, getRecentCard } from 'react-components/explore/explore.selectors.js';
+const validCountryId = 1;
+const validCommodityId = 2;
+const invalidCountryId = 1000;
+const invalidCommodityId = 1000;
+
+describe('Explore selectors', () => {
+  describe('getCardsWithRecentCard', () => {
+    const state = {
+      app: {
+        contexts: [
+          {
+            name: 'valid-context',
+            countryId: validCountryId,
+            commodityId: validCommodityId
+          }
+        ]
+      },
+      explore: {
+        sankeyCards: { data: [], meta: [] }
+      }
+    };
+
+    const selector = getCardsWithRecentCard;
+    beforeEach(() => {
+      selector.resetAllComputations;
+      jest.resetAllMocks();
+    });
+
+    const setMock = (countryId, commodityId) => jest.fn(() => `${countryId}-${commodityId}`);
+    it('returns recent card if it has a valid context', () => {
+        Storage.prototype.getItem = setMock(validCountryId, validCommodityId);
```

Comment:
Recommend adding this in the global jest config, so we don't have to care about local storage, 

```
class LocalStorageMock {
  constructor() {
    this.store = {};
  }

  clear() {
    this.store = {};
  }

  getItem(key) {
    return this.store[key] || null;
  }

  setItem(key, value) {
    this.store[key] = value.toString();
  }

  removeItem(key) {
    delete this.store[key];
  }
};

global.localStorage = new LocalStorageMock;
``` 

This is basically all you need to implement the necessary requirements for the API